jobs/tendrl-build-triggers.yml

```diff
@@ -100,7 +100,6 @@
     target_repo: 'tendrl-test'
     branch: 'release/1.4.0'
```

Comment:
I think its ok for now to update the release every time, However its good if we can clone automatically from the latest release branch on later point of time.